packages/world/test/World.t.sol

```diff
@@ -1645,12 +1645,11 @@ contract WorldTest is Test, GasReporter {
     world.registerNamespace(systemId.getNamespaceId());
     world.registerSystem(systemId, system, true);
 
-    string memory worldFunc = "testSelector()";
-    bytes4 sysFunc = WorldTestSystem.msgSender.selector;
+    string memory worldFunc = "msgSender()";
```

Comment:
should we call this `systemFunc`?

Reply:
or maybe just use the hardcoded string everywhere instead of a variable

Reply:
hardcoded strings would be more consistent i think

Reply:
but this variable is used more so will keep it for now